Applications/Spire/Include/Spire/KeyBindings/OrderTasksPage.hpp

```diff
@@ -0,0 +1,93 @@
+#ifndef SPIRE_ORDER_TASKS_PAGE_HPP
+#define SPIRE_ORDER_TASKS_PAGE_HPP
+#include <QWidget>
+#include "Spire/KeyBindings/OrderTask.hpp"
+#include "Spire/KeyBindings/OrderTasksRow.hpp"
+#include "Spire/KeyBindings/OrderTasksToTableModel.hpp"
+#include "Spire/Ui/TableView.hpp"
+
+namespace Spire {
+
+  /** Displays a page configuring key bindings for order tasks. */
+  class OrderTasksPage : public QWidget {
+    public:
+      
+      using Column = OrderTasksRow::Column;
+
+      /**
+       * Constructs a OrderTasksPage.
+       * @param region_query_model The model used to query region matches.
+       * @param model The list model of OrderTask.
```

Comment:
Avoid generic nouns as names. Instead of ```model``` which is used for a variety of nouns, rename this to ```order_tasks```.